python/mlcroissant/mlcroissant/_src/core/constants.py

```diff
@@ -39,6 +39,7 @@
 ML_COMMONS_SUB_FIELD = ML_COMMONS.subField
 ML_COMMONS_SUB_FIELD_TYPE = ML_COMMONS.SubField
 ML_COMMONS_TRANSFORM = ML_COMMONS.transform
+ML_COMMONS_OTHER_FIELD = ML_COMMONS.otherField
```

Comment:
Are RAI fields supposed to be nested in `otherField`?

Reply:
No, it's a mistake. But! I think RAI attributes should be grouped somehow, as they are an extension of the core vocabulary. For instance, if someone adds another extension (e.g., domain-specific for biology). How can the new attributes be grouped in the spec?  I guess the easy way is to provide a wrapper like:

`  "ReponsibleAI": 
    {
"dataCollection": "Text",
"dataCollectionType": "Text",
"dataCollectionTypeOthers": "Text",
 "dataCollectionMissingData": "Text",
[...]
    },`

And then other extensions should follow the same

`  "CellBiology": 
    {
      "attribute1": "...",
      "attribute2": "...",
     [...]
    },`

What do you think? If not, we can set it as plain fields, as the example field you added few days ago.